src/database/models/organization_item.rs

```diff
@@ -0,0 +1,361 @@
+use std::collections::HashMap;
+
+use crate::models::{
+    ids::base62_impl::{parse_base62, to_base62},
+    teams::ProjectPermissions,
+};
+
+use super::{ids::*, TeamMember};
+use redis::cmd;
+use serde::{Deserialize, Serialize};
+
+const ORGANIZATIONS_NAMESPACE: &str = "organizations";
+const ORGANIZATIONS_SLUGS_NAMESPACE: &str = "organizations_slugs";
+
+const DEFAULT_EXPIRY: i64 = 1800;
+
+#[derive(Deserialize, Serialize, Clone, Debug)]
+/// An organization of users who together control one or more projects and organizations.
+pub struct Organization {
+    /// The id of the organization
+    pub id: OrganizationId,
+
+    /// The slug of the organization
+    pub slug: String,
+
```

Comment:
org slugs and names should be the same, like usernames. it shouldn't be an additional field